dp-core/vr_interface.c

```diff
@@ -1974,8 +2018,12 @@ vr_interface_dump(vr_interface_req *r)
     vr_interface_req *resp = NULL;
     struct vr_interface *vif;
     struct vrouter *router = vrouter_get(r->vifr_vrf);
+    short core = r->vifr_core;
     struct vr_message_dumper *dumper = NULL;
 
+    if ((unsigned int)core > vr_num_cpus)
```

Comment:
I did it like this: if (r->vifr_core > 0 && r->vifr_core <= (int)vr_num_cpus) => sane; else => core = 0; The same in vr_interface_get().